internal/models/ekscluster/spec.go

```diff
@@ -25,9 +25,6 @@ type VmwareTanzuManageV1alpha1EksclusterSpec struct {
 	// EKS config for the cluster control plane.
 	Config *VmwareTanzuManageV1alpha1EksclusterControlPlaneConfig `json:"config,omitempty"`
 
-	// Nodepool definition for the cluster.
```

Comment:
Is the nodepool definition getting moved to some other place?

Reply:
Yes, we are removing it from the cluster's spec. All the nodepool operations to be done via nodepool's own APIs.

The field is still there in the API but is marked for deprecation, hence this MR.